birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -23,18 +23,30 @@ c.JupyterHub.db_url = '/persist/jupyterhub.sqlite'
 c.JupyterHub.template_paths = ['/custom_templates']
 
 class CustomDockerSpawner(DockerSpawner):
-
     def start(self):
         if(os.environ['MOUNT_IMAGE_SPECIFIC_NOTEBOOKS'] == 'true'):
-            host_dir = join(os.environ['JUPYTERHUB_USER_DATA_DIR'], 'tutorial-notebooks', self.user_options.get('image'))
-            image_dir = join('/notebook_dir/tutorial-notebooks', self.user_options.get('image'))
+            host_dir = join(os.environ['JUPYTERHUB_USER_DATA_DIR'], 'tutorial-notebooks')
+            image_dir = join('/notebook_dir/tutorial-notebooks')
+
+            #  Mount folder containing the notebooks that should be available to all images
+            self.volumes[join(host_dir, "common")] = {
```

Comment:
Just to be clear here, should I just remove the `common` dir volume here? Or should I also implement a way to easily customize the volume mounts for the specific images?
I don't think right now it is easy to have a `common` dir just for the eo/nlp images, since those volumes are handled by the jupyterhub_config.py.template, not by the env.local that we use to customize the environment in the daccs-configs repo. Unless this `common` folder is found directly in the specific image directory (ex.: tutorial-notebooks/crim-eo/common ), but it makes less sense.

We use the [ENABLE_JUPYTERHUB_MULTI_NOTEBOOKS](https://github.com/bird-house/birdhouse-deploy/blob/02d1e9db0cfc0ca0c2ca382e379228a48d4ca182/birdhouse/env.local.example#L228) variable right now to send a custom image whitelist from the env.local to the jupyterhub_config. So we could use a similar mechanic for custom volumes.

Anyway, let me know what I should do here, if I just remove the common volume, or if you need more.

Reply:
A solution was discussed with @dbyrns. We will put back a destination folder in the config.yml files, to add a way to customize the organization of a directory such as "tutorial-notebooks/crim-eo".
More to follow :)